execution/gethexec/executionengine.go

```diff
@@ -999,3 +999,12 @@ func (s *ExecutionEngine) Start(ctx_in context.Context) {
 		})
 	}
 }
+
+// Publish following functions for espresso caff node to access the blockchain
+func (s *ExecutionEngine) Bc() *core.BlockChain {
```

Comment:
Very small but could we choose a clearer name here? Although it just stands for Blockchain right?

Reply:
because this function is only to publish the private field `bc` and I think this name keeps consistency